app/containers/HomePage/index.js

```diff
@@ -116,5 +121,13 @@ const mapStateToProps = createStructuredSelector({
   error: makeSelectError(),
 });
 
-// Wrap the component to inject dispatch and state into it
-export default connect(mapStateToProps, mapDispatchToProps)(HomePage);
+const withMappedState = connect(mapStateToProps, mapDispatchToProps);
+
+const withReducer = injectReducer({ name: 'home', reducer });
```

Comment:
nit: should we maybe rename `name` to `key` to be more explicit? this is my fault for the initial implementation, would love to take this opportunity to fix it lol